packages/cli/src/utils/checks.ts

```diff
@@ -684,14 +720,18 @@ class CheckBuilder {
   // SANCTIONED_ADDRESSES is so well typed that if you call includes with a string it gives a type error.
   // same if you make it a set or use indexOf so concat it with an empty string to give type without needing to ts-ignore
   private readonly SANCTIONED_SET = {
-    data: new Set([''].concat()),
+    data: new Set<StrongAddress>(),
     wasRefreshed: false,
   }
 
+  private _formatAddress = (str: string) => ensureLeading0x(str.toLowerCase())
```

Comment:
why have the function be a class member at all? formatAddress could just be a free floating function the file

Reply:
I guess sure

Reply:
done in [1a8999b](https://github.com/celo-org/developer-tooling/pull/610/commits/1a8999bc553747ce6775cd1f35923ad53c41a0ad)